README.md

```diff
@@ -85,6 +85,27 @@ necessary.
 
 When parsing only one markdown file, the title of the deck could be generated based on the top-level headline (`#`).
 
+To embed cards definitions in the existing document, one can limit the scope of the document to be processed between one or more matching pairs of
```

Comment:
I would add a ### heading here. Maybe called “Embedding” or something, so it’s its own section